docs/operator-docs/update-cumulus-dependencies.md

```diff
@@ -0,0 +1,70 @@
+---
+id: update-cumulus-dependencies
+title: Update Cumulus Dependencies
+hide_title: true
+---
+
+# Updating Cumulus Dependencies
+
+This document will outline, step by step, how to update a Cumulus deployment to a newer version. It is currently assumed that _all_ Cumulus packages are to be updated to the same version at the same time. This is expected to change in the future.
+
+## Check Current Version
+
+In the directory from which you deploy Cumulus (the directory that has `app/` and `iam/` sub-directories), run the following:
+
+```shell
+$ yarn list --depth 0 | grep "@cumulus" | awk '{ print $2 }'
+@cumulus/api@1.11.0
+@cumulus/common@1.11.0
+@cumulus/deployment@1.11.0
+...
+@cumulus/test-processing@1.11.0
+```
+
+The command above lists the Cumulus packages and their version in the format `@cumulus/<package>@<version>`. Note that `<version>` is your current version.
+
+## Update to Latest Version
+
+One can update packages fairly painlessly using the provided `yarn upgrade` command.
+
+To upgrade all Cumulus to the latest Cumulus version:
+
+```shell
+$ yarn upgrade --latest --pattern "@cumulus/*"
+```
+
+To upgrade an individual Cumulus `<package>` to the latest version:
+
+```shell
+$ yarn upgrade --latest @cumulus/<package>
+```
+
+## Update to a Specific Version
+
+To upgrade individual Cumulus packages to a specific `<version>` is fairly simple: For each package,
+
+```shell
+$ yarn upgrade @cumulus/<package>@<version>
+```
+
+*Note:* `<version>` is a version string that observes the syntactic rules outlined [here](https://docs.npmjs.com/files/package.json#dependencies).
+
+
+## Redeploy
+
+Note that there may be configuration changes required with a new version. Please check the [release documentation](https://nasa.github.io/cumulus/docs/upgrade/upgrade-readme) for instructions regarding deployment/stack configuration. Once configuration is in an acceptable state, deployment commands should be the same as before.
```

Comment:
I would link to the release documentation on github instead of the upgrade instructions, since they are linked below.